api/types/types.go

```diff
@@ -270,7 +258,7 @@ type ContainerJSONBase struct {
 	HostnamePath    string
 	HostsPath       string
 	LogPath         string
-	Node            *ContainerNode `json:",omitempty"` // Node is only propagated by Docker Swarm standalone API
+	Node            *ContainerNode `json:",omitempty"` // Deprecated: Node was only propagated by Docker Swarm standalone API. It sill be removed in the next release.
```

Comment:
nit: "will be removed". 
Also thinking, should we specify an expected version in this comment, just to avoid confusion for people down the line if we forget to remove it?